confgenerator/testdata/valid/linux/metrics-receiver_rabbitmq_no_tls/input.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+metrics:
+  receivers:
+    rabbitmqmetrics:
```

Comment:
`s/rabbitmqmetrics/rabbitmq` and `s/rabbitmqpipeline/rabbitmq` here? Similar for other files. See more context in https://github.com/GoogleCloudPlatform/ops-agent/pull/384#issue-1123802764

Also can we ensure there is a `confgenerator/testdata/valid/linux/metrics-receiver_rabbitmq/golden_fluent_bit_main.conf` test case that verifies the bare minimum of the configuration (aka only the required configuration parameters are present). `collection_interval` is optional for instance.

Reply:
Updated

Reply:
Looks like there are still some files with `rabbitmqmetrics` and `rabbitmqpipeline`.